internal/client.go

```diff
@@ -391,7 +391,7 @@ type (
 		// Optional: defaulted to a uuid.
 		ID string
 
-		// TaskQueue - The decisions of the workflow are scheduled on this queue.
+		// TaskQueue - The workflow tasks of the workflow are scheduled on this queue.
```

Comment:
s/on this queue/on the queue with this name
s/This is also the default task queue on which activities are scheduled./This is also the name of the activity task queue on which activities are scheduled./